internal/config/colorscheme.go

```diff
@@ -81,9 +81,10 @@ func LoadColorscheme(colorschemeName string) error {
 // Colorschemes are made up of color-link statements linking a color group to a list of colors
 // For example, color-link keyword (blue,red) makes all keywords have a blue foreground and
 // red background
-func ParseColorscheme(text string) (map[string]tcell.Style, error) {
+func ParseColorscheme(name string, text string) (map[string]tcell.Style, error) {
 	var err error
-	parser := regexp.MustCompile(`color-link\s+(\S*)\s+"(.*)"`)
+	colorParser := regexp.MustCompile(`color-link\s+(\S*)\s+"(.*)"`)
+	includeParser := regexp.MustCompile(`include:\s+"(.*)"`)
```

Comment:
`include` seems better than `include:` (more consistent with `color-link`).

Reply:
Yes, from this perspective it differs. It was inspired by the YAML syntax approach. But since it's no YAML but *.micro instead we can do this to have it aligned. :)

Reply:
I agree with @dmaluka, nothing else in the color scheme file format uses colons so it's a weird inconsistency to have. In my mind a colon implies a key-value structure (like a JSON object or Python dict, with unique keys) which doesn't match semantics for `include` (you can have multiple `include` statements to combine colorschemes).

Reply:
I already removed the `:` in front of the `include`

The more tricky part is the...
> (you can have multiple include statements to combine colorschemes)

...which might end up in circular includes. One possible solution is to allow only one level of include or maybe a bit smarter to store the name of the initial one and don't allow the circle.

Reply:
> I already removed the `:` in front of the `include` (locally)

Update also readme